### DISCUSSION TITLE
Definitions.Default.All() doesn´t exist in the package

### DISCUSSION THREAD

**Discussion by raphaelmoreira** [MediatedCommunications/Mime-Detective#29] [Q&A] [answered]
Hello.

I'm trying to use the package in a .net framework (4.7.2), with asp.net mvc 4. I had installed Mime-Detective only and tried:

```
using MimeDetective;
var Inspector = new ContentInspectorBuilder() {
    Definitions = Definitions.Default.All()
}.Build();
```

`Error: Definitions.Default.All() doesn´t exist in the package.`

I don´t know what I did wrong.

**Comment by TonyValenti** [comment, marked as answer]
Hi @raphaelmoreira -
You need to fully qualify Definitions.
```MimeDetective.Definitions.Default.All()```

**Reply by raphaelmoreira** [reply]
Hello, Tony.

Even with `using MimeDetective;` directive?

This was unexpected. I believe it is necessary to change the documentation, as this is not mentioned in the example.

Anyway, thanks.

**Reply by TonyValenti** [reply]
Yes!  You're right.  I updated the docs.
